v0.3/tutorials/quickstart.eve

```diff
@@ -7,117 +7,95 @@ title: Quickstart
 
 # Quickstart
 
-This guide is a 5 - 10 minute introduction to the essential concepts in Eve. If you've never used Eve before, you're in the right place! Before you start with this tutorial, please follow the [installation and usage](/v0.3/install/) instructions, which will get you running Eve programs on your machine. In the eve-starter/programs directory, create an empty document called "quickstart.eve". Use your favorite editor to edit the program, and run it with the command:
+This guide will give you a quick introduction to Eve by making a simple webapp in just a few minutes.
 
-```
-npm start -- eve-starter/programs/quickstart.eve
-``` 
-
-Eve documents are written in Markdown. Eve code is written in blocks, delineated by Markdown code fences (matching pairs of ``` or ~~~ surrounding a section of code). These blocks of code search for and create records, which are key value pairs attached to a unique ID.
+## Getting Eve Running
 
-## Adding records to Eve
+Before you start with this tutorial, please follow the [installation and usage instructions](/v0.3/install/), which will get you running Eve programs on your machine. In the eve-starter directory, you can launch the Program Switcher:
 
-Let's start with a block that adds a record to Eve (we'll show the code fence in this block, but elide them in later blocks):
+```
```

Comment:
This is still an issue. Otherwise the docs highlighter will think this block is meant to be eve code.